src/openeo_aggregator/backend.py

```diff
@@ -160,18 +172,58 @@ def _merge_collection_metadata(self, by_backend: Dict[str, dict]) -> dict:
                             or [[None, None]],
             },
         }
-        result["links"] = list(getter.union("links"))
+        # links
+        result["links"] = [l for l in list(getter.union("links")) if l["rel"] not in ("self", "parent", "root")]
+        result["links"].append({
+            "href": url_for("openeo.collections", _external=True),
+            "rel": "root"
+        })
+        result["links"].append({
+            "href": url_for("openeo.collections", _external=True),
+            "rel": "parent"
+        })
+        result["links"].append({
+            "href": url_for("openeo.collection_by_id", collection_id=result["id"], _external=True),
+            "rel": "self"
+        })
+        # cube_dimensions
         # TODO: combine cube:dimensions smarter?
         cube_dimensions = getter.first("cube:dimensions")
         if cube_dimensions:
             result["cube:dimensions"] = cube_dimensions
-        # TODO merge existing summaries?
-        result["summaries"] = {
-            # TODO: use a more robust/user friendly backend pointer than backend id (which is internal implementation detail)
-            self.STAC_PROPERTY_PROVIDER_BACKEND: list(by_backend.keys())
-        }
-        # TODO: assets ?
-
+        # summaries
+        # TODO: Better merging for summaries?
+        for summary in getter.get("summaries"):
+            print(summary)
+        result["summaries"] = {}
+        # TODO: use a more robust/user friendly backend pointer than backend id (which is internal implementation detail)
+        result["summaries"][self.STAC_PROPERTY_PROVIDER_BACKEND] = list(by_backend.keys())
+        # assets
+        # TODO: assets ? For now, take the union.
+        result["assets"] = list(getter.union("assets"))
+
+        # crs
```

Comment:
what is this crs field? It's not defined in openeo spec.

Reply:
It was mentioned that it was missing from the aggregator in this issue:
https://github.com/openEOPlatform/architecture-docs/issues/260

It is included in e.g. the [SENTINEL1_GRD](https://openeo.vito.be/openeo/collections/SENTINEL1_GRD) collection on the VITO backend. But I am unable to find the STAC extension that requires the field. So I will exclude it from the aggregator metadata for now.

Reply:
raised the question here https://github.com/openEOPlatform/architecture-docs/issues/260#issuecomment-1225595718

Reply:
based on the response https://github.com/openEOPlatform/architecture-docs/issues/260#issuecomment-1225610441, can you add some pointers in that `# crs`  comment?

Reply:
Done.